.drone.yml

```diff
@@ -7354,7 +7355,7 @@ steps:
   - docker buildx build --push --builder "teleport-operator-v15-arm64-builder" --platform
     "linux/arm64" --tag drone-docker-registry:5000/teleport-operator:$(cat "/go/var/full-version")-arm64
     --file "/go/src/github.com/gravitational/teleport/integrations/operator/Dockerfile"
-    --build-arg BUILDBOX=public.ecr.aws/gravitational/teleport-buildbox-arm:teleport15
+    --build-arg BUILDBOX=public.ecr.aws/gravitational/teleport-buildbox-arm:teleport15-amd64
```

Comment:
I figured this would be confusing, so I left a comment in `dronegen/buildbox.go`. See https://github.com/gravitational/teleport/pull/34950/files#diff-e70d5a477a1e56e0b2f65e4b4dc97e99e870b7ea253572d8e8994027dac6a8eaR69.

Is that good enough, or do you want something in `.drone.yml` as well (not sure how possible this is, as everything changed in `.drone.yml` is from dronegen)?

Reply:
oh, and see my comment addition in `dronegen/container_image_products.go` as well.